casper/src/main/resources/PoS.rhox

```diff
@@ -344,17 +405,25 @@ new  PoS,
           }
         } |
 
-        contract deposit (@fromUserPk, @amount, returnCh) = {
+        contract deposit (@fromUserPk, @amount, @activeValidators, returnCh) = {
           new vaultCh,
               revAddressCh,
+              indexSender,
+              getBlockData(`rho:block:data`), blockDataCh,
               authKeyCh in {
             @RevVault!("deployerAuthKey", *authKeyCh) |
+            getBlockData!(*blockDataCh) |
```

Comment:
This code would be more readable if the send on `getBlockData` was in the same lexical scope as the receive on `getBlockData`.